ipn/ipnlocal/local.go

```diff
@@ -6449,13 +6449,36 @@ func (n lastSuggestedExitNode) asAPIType() (res apitype.ExitNodeSuggestionRespon
 	return res, ErrUnableToSuggestLastExitNode
 }
 
+// processAllowedCandidates converts a string slice of exit node candidate IDs to a map of strings and ints. Int value represents nothing and won't be used.
+// Used for a faster lookup comparison in suggestExitNode.
+func processAllowedCandidates(allowedCandidates []string) (candidates map[string]int) {
+	candidates = make(map[string]int, len(allowedCandidates))
+	for _, candidate := range allowedCandidates {
+		candidates[candidate] = 0
+	}
+	return candidates
+}
```

Comment:
this is just set.Of ?

see the set package

Reply:
I'm pretty sure that `set.Of` will work great here.